meson_options.txt

```diff
@@ -16,6 +16,9 @@ option('use_png', type : 'boolean', value : true,
 option('use_slirp', type : 'boolean', value : true,
        description : 'Enable Ethernet emulation using Libslirp')
 
+option('enable_tracy', type : 'boolean', value : false,
```

Comment:
In a couple Meson wrap PRs, I had similar boolean-prefixed names, like:
 - `enable_unittests`
 - `with_examples`
 - `build_demos`

The feedback from the Meson team on all of them was to eliminate the boolean prefix and go with more noun-like names: `'unittests'`, `'demos`', and `'examples'`.

Applying that here, I think going with `'tracy'` is a nice fit: `-Dtracy=true` looks a bit better than  `enable_tracy=true`.

I know that's a bit of departure from our existing meson options, but we didn't have that guidance from the Meson folks at the time.  So I think we can start moving our options to the noun-style going forward (perhaps even a mass-cleanup, at some point).